src/lab_4.py

```diff
@@ -0,0 +1,74 @@
+class Shoes:
+    #створюємо клас 
+    public_number = 0 # числове поле 
+    public_string = "String" # стрічкове поле   
+    def __init__(self, name="Default Name", price=0, size=0, colour="Unknown Colour"):
+        #init - ініціалізація(налаштувати при створенні), надати початкові значення 
+        #self - перший параметр ,який надає  доступ до атрибутів 
+        #self - це параметр, що передає посилання на поточний об'єкт класу, дозволяючи взаємодіяти з його полями.
+        # приватній поля 
```

Comment:
those comments look like generated by chatGPT

Reply:
my miss , i remove this comments